scripts/build_pycer.sh

```
@@ -2,8 +2,31 @@
 
 # Rebuild pycer Python bindings
 # Forces a fresh build from C++ source
+#
+# Usage:
+#   scripts/build_pycer.sh              # Normal build
+#   scripts/build_pycer.sh -s address   # Build with address sanitizer
+#   scripts/build_pycer.sh -s thread    # Build with thread sanitizer
 
 cd "$(dirname "$0")"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)